qmk_firmware/keyboards/preonic/keymaps/that_canadian/Makefile

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
This file should be deleted.

Reply:
yes, i honestly don't know how this got in my repo.